pkg/cli/common.go

```diff
@@ -280,3 +281,12 @@ func DefaultIsolation() string {
 	}
 	return buildah.OCI
 }
+
+func VerifyFlagsArgsOrder(args []string) error {
+	for _, arg := range args {
+		if strings.HasPrefix(arg, "-") {
+			return errors.Errorf("%s should be set before the image name", arg)
```

Comment:
Perhaps make this a little more informative.

"No options (%s) can be specified after the image or container name"

Reply:
Yep, I like this better. Thanks!